src/utilities/validate.ts

```diff
@@ -0,0 +1,8 @@
+export function isJson(str: any) {
+	try {
+		JSON.parse(str);
+	} catch (e) {
+		return false;
```

Comment:
should we handle the exception here?

Reply:
yeah, because if the parse fails it throws an error so to handle it we need try/catch, it will help to understand if the user actually provided correct data